test/jdk/javax/swing/ImageIcon/LoadInterruptTest.java

```diff
@@ -33,7 +33,8 @@
 /*
  * @test
  * @bug 8236987
- * @summary Verifies ImageIcon constructor produces no output when the thread is interrupted
+ * @summary Verifies ImageIcon constructor produces no output when the
+ * thread is interrupted
```

Comment:
```suggestion
 * @summary Verifies ImageIcon constructor produces no output when the
 *          thread is interrupted
```
I believe this improves the readability of the summary and it's usually formatted like this.